std_msgs/Header header
float32 temperature   #Centigrade
float32 rh            #relative humidity %RH
float32 smoke         #ppm
float32 pm1_0         #ug/m^3
float32 pm2_5         #ug/m^3
float32 pm10          #ug/m^3
float32 illuminance   #lx
float32 noise         #db

